### ISSUE TITLE
Redirection Plugin Error

### ISSUE THREAD

**Issue by mahmudulhrabby** [johngodley/redirection#2432]
Hi There, Thanks for creating Helpful the Plugin.
![Screenshot](https://user-images.githubusercontent.com/33920270/91364607-f4b01780-e820-11ea-8274-410b073ae4ba.png)

I've got error from long time.
I can't use the Plugin. Not all features are working
**Screenshot** https://prnt.sc/u6i34i

Please give a solution.

Best Regards,
Rabby

```
Plugin: 4.8
WordPress: 5.5 (single)
PHP: 7.4.9
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.135 Safari/537.36
JavaScript: https://assets.pixeup.com/plugins/redirection/redirection.js
REST API: https://pixeup.com/wp-json/

Error: No route was found matching the URL and request method (rest_no_route)
Raw: {"code":"rest_no_route","message":"No route was found matching the URL and request method","data":{"status":404}}
```

**Comment by johngodley** [comment]
As the message indicates this could be caused by a security plugin. It could also be because your site has different URLs - pixeup.com and assets.pixeup.com - these should really be the same domain.

**Comment by mahmudulhrabby** [comment]
Hi Jhon, Thanks for your quick reply. and sorry for writing same topic in 2 places.

Yes, I used sub-domain website resources(images, files). Because It helps me for website performance and aslo it will help me in future for many type of work, such as easy CDN setup.

**I don't think different URLs is a problem.** Because I used domain and sub-domain for the same site from a long time and I used Your Redirection plugin from a long time including in multiple sites, which is the same as this site. All was well. This problem has started for less then 1 month.

I have tried to turn off some security-related plugin to fix the issue. But it didn't work.

let me know your opinion.

Best Regards,
Rabby
